basic_ingest.install

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @file
+ * Update functions for the Basic Ingest module.
+ */
+
+/**
+ * Moves the help text to config.
+ */
+function basic_ingest_update_9001() {
+  $help_text = <<<EOHELP
+ <p>Adding this Repository Item to a collection isn't required, but can be used to organize items for display purposes. Use the 'Add container' button to facilitate multiple collection inheritance.</p>
+ <p>When creating a Repository Item using the 'Collection' content type, you will also be asked to optionally provide a 'Representative Image'; this will be used as a thumbnail, as well as in other situations where an image representative of the collection may be desirable.</p>
+ <p>By default, if any Media types are associated with the selected content type, you will be redirected to the Media ingest page after clicking 'Save'.</p>
+ <p>Uncheck 'Published' to keep this Repository Item out of public view. Unpublished items will still be preserved.</p>
+ EOHELP;
+
+  $current_help_text = \Drupal::config('node.type.islandora_object')
+    ->get('help') ?: '';
+  \Drupal::service('config.factory')->getEditable('node.type.islandora_object')
+    ->set('help', $help_text . PHP_EOL . $current_help_text)
```

Comment:
This still makes use of the wrong line-ending sequence, as mentioned in https://github.com/discoverygarden/basic_ingest/pull/26#discussion_r2325706830